src/types.ts

```diff
@@ -66,11 +65,7 @@ export interface Ctx {
   _random?: {
     seed: string | number;
   };
-  // enhanced by events plugin
   events?: EventsAPI;
-  // enhanced by player plugin
-  player?: PlayerAPI;
-  // enhanced by random plugin
   random?: RandomAPI;
 }
```

Comment:
Based on @delucis comments
> The Events & Random plugins are always included automatically in every game set-up, so these would be the default plugins

Tried making `events` and `random` not optional but it break 20 or so odd tests so I left it as optional. Perhaps our assumption was incorrect

Reply:
Ah, my guess would be that this is because there are actually two different contexts: a POJO that is serialised and saved as state/transferred over web sockets, and an “enhanced” context that has the plug-in API methods attached. There’s probably a need internally to go through marking what’s using the plain context and what’s using the enhanced context.

Reply:
Got it, I won't include in this PR but I think two interface might make sense then. Maybe `InternalCtx` and `Ctx`